src/main/java/seedu/partyplanet/logic/parser/EListCommandParser.java

```diff
@@ -140,7 +144,7 @@ public EListCommand parse(String args) throws ParseException {
     private Comparator<Event> applySortDirection(
             Comparator<Event> comparator, ArgumentMultimap argMap) throws ParseException {
         Optional<String> orderType = argMap.getValue(PREFIX_ORDER);
-        if (orderType.isEmpty()) {
+        if (orderType.isEmpty() || comparator == SORT_EVENTDATE_UPCOMING) {
```

Comment:
This makes sort_upcoming unable to work with asc/desc.
Isit intented? If so then just remember to document
Same for `list`

Reply:
Yes, very much intended, because sorting in "reverse upcoming" doesn't seem to have a use case. Will be documenting this eventually.

Think it's a good idea to support it?

Reply:
Make sense! just making sure it isnt an oversight